src/sql/workbench/common/actions.ts

```diff
@@ -253,7 +253,7 @@ export class BackupAction extends Task {
 		}
 		if (profile) {
 			const serverInfo = connectionManagementService.getServerInfo(profile.id);
-			if (serverInfo && serverInfo.isCloud) {
+			if (serverInfo && serverInfo.isCloud && profile.providerName.toLowerCase() === 'mssql') {
```

Comment:
Use mssqlProviderName from src\sql\platform\connection\common\constants.ts instead

And you can just compare against the value directly - not necessary to lower case.

Reply:
fixed